fda-model/metrics/AlternativeResults.cs

```diff
@@ -50,24 +51,48 @@ public AlternativeResults()
             _aaeqResults = new ConsequenceDistributionResults();
             BaseYearScenarioResults = new ScenarioResults();
             FutureYearScenarioResults = new ScenarioResults();
+            AnalysisYears = new List<int>() { 2030, 2049 };
```

Comment:
Are these meant to be hardcoded?

Reply:
Yes, this is the "null object" we return if we can't find the requested results or if we can't compute the requested results.  I thought I could avoid potential null ref exceptions if we at least have dummy parameter values like we do with the distributions.